src/Gridify/GridifyGlobalConfiguration.cs

```diff
@@ -48,6 +47,16 @@ public static class GridifyGlobalConfiguration
       /// </summary>
       public static bool DisableNullChecks { get; set; } = false;
 
+      /// <summary>
+      /// Specifies how field names are inferred from CLR property names.
+      /// By default, Elastic.Clients.Elasticsearch uses camel-case property names.
+      /// </summary>
+      /// <example>
+      /// If false CLR property EmailAddress will be inferred as "emailAddress" Elasticsearch document field name
+      /// If true, the CLR property EmailAddress will be inferred as "EmailAddress" Elasticsearch document field name
+      /// </example>
+      public static Func<string, string>? CustomElasticsearchNamingAction { get; set; }
```

Comment:
_**No Action is needed.**_
for now, it is okay to have this setting here but ideally would be better if we find another way to keep other extensions's configurations separated. (Just said this in case you could find a cleaner solution for this problem)